core/modules/admin_bar/css/admin_bar.css

```diff
@@ -326,6 +326,11 @@
   padding: 0.5em 2em 0.5em 1em; /* LTR */
   width: 100%;
   box-sizing: border-box;
+  /* Reinforce the admin bar font so the search input is unaffected by the
+  theme. */
+  font: inherit;
+  line-height: 1.15;
+  font-family: sans-serif;
```

Comment:
The font is already inherited, so setting this again overrides `"Lucida Grande",Verdana,...`, which it would normally be for admin bar to "sans-serif".

Reply:
@indigoxela 

On a site without a theme that affects it (e.g. Basis) then the admin bar search input is actually getting the font family from `core/misc/normalize.css` from this selector:

```css
button, input, optgroup, select, textarea {
    font-family: sans-serif;
    font-size: 100%;
    line-height: 1.15;
    margin: 0;
}
```
The `margin: 0;` line is overridden.

<img width="1637" height="800" alt="image" src="https://github.com/user-attachments/assets/71297cf7-2f2a-4694-b6c7-fa9c51b0e283" />

Reply:
Hm...

If this is getting too difficult in terms of "which font should apply" and "what people might expect", then just mark this as resolved. I don't have strong opinions and can override it in my themes, anyway. :shrug: